lib/services/doctor-service.ts

```diff
@@ -26,96 +26,114 @@ class DoctorService implements IDoctorService {
+						+ `To be able to build such projects, verify that you have at least ${DoctorService.MIN_SUPPORTED_POD_VERSION} version installed.`);
+					result = true;
+				}
+			} else {
+				this.$logger.out("NOTE: You can develop for iOS only on Mac OS X systems.");
+				this.$logger.out("To be able to work with iOS devices and projects, you need Mac OS X Mavericks or later." + EOL);
 			}
 
-			if (sysInfo.cocoapodVer && semver.valid(sysInfo.cocoapodVer) === null) {
-				this.$logger.warn(`WARNING: CocoaPods version is not a valid semver version.`);
-				this.$logger.out("You will not be able to build your projects for iOS if they contain plugin with CocoaPod file." + EOL
-					+ `To be able to build such projects, verify that you have at least ${DoctorService.MIN_SUPPORTED_POD_VERSION} version installed.`);
-				result = true;
+			let androidToolsIssues = this.$androidToolsInfo.validateInfo().wait();
+			let javaVersionIssue = this.$androidToolsInfo.validateJavacVersion(sysInfo.javacVersion).wait();
+			let doctorResult = result || androidToolsIssues || javaVersionIssue;
+
+			if (!configOptions || configOptions.trackResult) {
+				this.$analyticsService.track("DoctorEnvironmentSetup", doctorResult ? "incorrect" : "correct").wait();
 			}
 
-			if (sysInfo.cocoapodVer && semver.valid(sysInfo.cocoapodVer) && semver.lt(sysInfo.cocoapodVer, DoctorService.MIN_SUPPORTED_POD_VERSION)) {
-				this.$logger.warn(`WARNING: CocoaPods version is lower than ${DoctorService.MIN_SUPPORTED_POD_VERSION}.`);
-				this.$logger.out("You will not be able to build your projects for iOS if they contain plugin with CocoaPod file." + EOL
-					+ `To be able to build such projects, verify that you have at least ${DoctorService.MIN_SUPPORTED_POD_VERSION} version installed.`);
-				result = true;
+			if (doctorResult) {
+				this.$logger.info("There seem to be issues with your configuration.");
+				if (this.$hostInfo.isDarwin) {
+					this.promptForHelp(DoctorService.DarwinSetupDocsLink, DoctorService.DarwinSetupScriptLocation, []).wait();
+				} else if (this.$hostInfo.isWindows) {
+					this.promptForHelp(DoctorService.WindowsSetupDocsLink, DoctorService.WindowsSetupScriptExecutable, DoctorService.WindowsSetupScriptArguments).wait();
+				} else {
+					this.promptForDocs(DoctorService.LinuxSetupDocsLink).wait();
+				}
 			}
-		} else {
-			this.$logger.out("NOTE: You can develop for iOS only on Mac OS X systems.");
-			this.$logger.out("To be able to work with iOS devices and projects, you need Mac OS X Mavericks or later." + EOL);
-		}
 
-		let androidToolsIssues = this.$androidToolsInfo.validateInfo().wait();
-		let javaVersionIssue = this.$androidToolsInfo.validateJavacVersion(sysInfo.javacVersion).wait();
-		let doctorResult = result || androidToolsIssues || javaVersionIssue;
+			let versionsInformation: IVersionInformation[] = this.$versionsService.getComponentsForUpdate().wait();
```

Comment:
Please wrap this code in `try-catch`, so in case we are unable to get latest version from npm, `tns doctor` will not fail.